docs/source/analyze/outcomes.md

```diff
@@ -33,3 +33,16 @@ When the uri attribute is present, a small "Read More" icon will be displayed on
 ## When
 
 Some analyzers implement the "when" attribute. The details and implementation of this attribute vary between analyzers. For example, the [cluster version](https://troubleshoot.sh/docs/analyze/cluster-version/) analyzer uses this as a semver comparator, while the [image-pull-secrets](https://troubleshoot.sh/docs/analyze/image-pull-secrets/) analyzer doesn't need the when, it's a simple, binary output.
+
+The `when` attribute can be used with the operators `=`, `==`, and `===` in expression language. The value to the left of the operator should be a variable. The value to the right of the operator should be a data type, such as a string or number.
+
+Typically, these operators function as follows:
+
+---
+| Assignment Operator | Description|
+| ----- | ---------------------------------|
+| = | Functions as an assignment operator that sets the value on the left of `=` to the value on the right.|
```

Comment:
All three operators are used for comparison and there is no difference between them.  For example https://github.com/replicatedhq/troubleshoot/blob/73609c4feff5760ce59f58b124dcc3196a0172ae/pkg/analyze/ceph.go#L183

Reply:
@divolgin  Please check my changes and see if it's ok to deploy. Thanks for your help!